packages/setcore/setcore.1.0.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+authors: "Roberto Di Cosmo <roberto@dicosmo.org>"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/setcore"
+dev-repo: "git+https://github.com/UnixJunkie/setcore.git"
+bug-reports: "https://github.com/UnixJunkie/setcore/issues"
+build: [
+  ["autoconf"]
+  ["autoheader"]
+  ["./configure"]
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "conf-autoconf"
+  "ocaml"
+  "ocamlfind"
```

Comment:
are you sure you need ocamlfind? How is it required in the build process, I can only see it in configure but no ocaml libraries are needed

Reply:
At the very least if it's really needed can you tag it as a `build` dependency too?

Reply:
Yes, ocamlfind is just some left-over piece in the configure things.
I will remove it and retag the software.